packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -192,6 +192,24 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     }
   }
 
+  // Reads title and lead from the first block and saves them in variables
+  function syncBlocksWithMeta() {
+    const titleBlock = blocks.map(block => {
```

Comment:
I think ti's better if you use array.find here. The `find` method returns the first element that fits the description. If there is n title block it will be undefined.